src/Microsoft.Data/PrimitiveColumn.cs

```diff
@@ -145,10 +147,9 @@ protected override object GetValue(long startIndex, int length)
             return _columnContainer[startIndex, length];
         }
 
-        protected override object GetValue(long rowIndex)
-        {
-            return _columnContainer[rowIndex];
-        }
+        internal T? GetTypedValue(long rowIndex) => _columnContainer[rowIndex];
```

Comment:
What's the difference between this method and the `T? this[long rowIndex]` indexer?

Reply:
None. I initially thought it wasn't worth refactoring since it's just 1 line. The indexer now calls `GetTypedValue`. `GetTypedValue` itself exists because I want to pass the method as a delegate to other functions